scripts/error-codes/dev-expression-with-codes.js

```diff
@@ -117,16 +117,6 @@ module.exports = function(babel) {
             if (prodErrorId === undefined) {
               // The error cannot be found in the map.
               node[SEEN_SYMBOL] = true;
-              if (process.env.NODE_ENV !== 'test') {
```

Comment:
@tomocchino Just change this to `if (process.env.NODE_ENV === 'publish') {`. There's no spec for this. Only convention. I just made up this one. We don't actually have to start using it yet because our publishing strategy is in flux.